recipes/gargammel/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+sed -i  's/make -C libgab/make -C libgab CXX=$(CC)/' Makefile
```

Comment:
On macOS, the `-i` option to `sed` works a bit differently and you need to write `sed -i.bak` instead (this also works on Linux).

Reply:
Hi Marcel, thanks for the tip and the review (s). I will fix once I get this to actually compile properly on Linux....

I'm having issues with the fact there is 'compilation with compilation' (ugly, but not my decision) and the C++ variables for paths aren't inherited into the internal make files.

Reply:
I can understand your pain. For these types of tools that download their dependencies at build time, packaging is a bit annoying. The cleanest way to handle this would be to patch the tool so it can work with already-installed dependencies, and list those in the recipe. But this is only a suggestion since it sometimes a bit hard to achieve and many other recipes aren’t perfect in that regard either.